examples/c/nonlinear_chain_ocp_nlp.c

```diff
@@ -1150,6 +1154,11 @@ int main()
 		else
 		{
 			plan->nlp_dynamics[i] = DISCRETE_MODEL;
+			if (NMF > 3) // in this case the discrete model is not supported, commented in cmake to speed up compilation
+			{
+				plan->nlp_dynamics[i] = CONTINUOUS_MODEL;
+				plan->sim_solver_plan[i].sim_solver = IRK;
```

Comment:
better throw an error and exit instead of changing the option  to something else.